samples/nrf9160/asset_tracker/README.rst

```diff
@@ -71,6 +72,7 @@ The sample is built as a non-secure firmware image, thus with ``CONFIG_TRUSTED_E
 It can be programmed independently from the secure boot firmware.
 
 See :ref:`gs_programming` for information about how to build and program the application.
+The sample has a Kconfig file with options to configure the sample, such as enabling power optimization and selecting simulated or real sensor data. These can be browsed and configured using the command line tool ``menuconfig`` or directly in ``prj.conf``.
```

Comment:
```suggestion
The Kconfig file of the sample contains options to configure the sample.
For example, configure :option:`CONFIG_POWER_OPTIMIZATION_ENABLE` to enable power optimization or :option:`CONFIG_TEMP_USE_EXTERNAL` to use an external temperature sensor instead of simulated temperature data. 
In |SES|, select **Project** > **Configure nRF Connect SDK project** to browse and configure these options.
Alternatively, use the command line tool ``menuconfig`` or configure the options directly in ``prj.conf``.
```